lib/options.py

```diff
@@ -201,135 +286,190 @@ def __init__(self):
                 nargs='?',
                 default=env_int('TEST_RUN_JOBS', 0),
                 type=int,
-                help="""Workers count. Default: ${TEST_RUN_JOBS} or 0 (0 means
-                2 x CPU count). -1 means everything running consistently
-                (single process). """)
+                help=format_help(
+                    """
+                    Workers count.
+
+                    0 means 2 x CPU count.
+                    -1 means everything running consistently (single process).
+
+                    Default: ${TEST_RUN_JOBS} or 0.
+                    """))
 
         parser.add_argument(
                 "-r", "--retries",
                 dest='retries',
                 default=env_int('TEST_RUN_RETRIES', 0),
                 type=int,
-                help="""The number of test run retries after a failure.
-                Default: ${TEST_RUN_RETRIES} or 0. It is also the default value
-                for 'fragile' tests unless the `retries` option is set in the
-                suite.ini config file.""")
+                help=format_help(
+                    """
+                    The number of test run retries after a failure.
+
+                    Default: ${TEST_RUN_RETRIES} or 0. It is also the default
```

Comment:
Let's move `Default: ${TEST_RUN_RETRIES} or 0.` to the end like it's done everywhere.

Reply:
Done. Resolving the discussion as obviously fixed.